src/components/Dragons/Dragons.js

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import DragonCard from './DragonCard';
 
-const Dragons = () => (
-  <div>Dragons</div>
-);
+const Dragons = () => {
+  const state = useSelector((state) => state.dragonSlice);
+
+  return (
+    <>
+      <div>
+        {state.map((dragon) => (
+          <DragonCard
+            key={dragon.id}
+            id={dragon.id}
+            name={dragon.name}
+            src={dragon.img}
+            type={dragon.type}
+            value={dragon.value}
+          />
+        ))}
+
+      </div>
+    </>
+  );
+};
```

Comment:
Here there is some errors related to the Dragons component when trying to make the map of something that is undefined

![image](https://github.com/victormc13/Space-Travelers-Hub/assets/110580299/74fefe43-b309-411c-97cb-f140eb31dd25)

Reply:
done